lib/Dialect/ESI/ESITranslations.cpp

```diff
@@ -91,11 +95,11 @@ LogicalResult ExportCosimSchema::visitEndpoint(CosimEndpointOp ep) {
   if (epName)
     os << epName << " endpoint at " << ep.getLoc() << ":\n";
   os << "#   Send type: ";
-  sendTypeSchema.writeMetadata(os);
+  sendTypeSchema->writeMetadata(os);
```

Comment:
A toplevel function like (ostream, Schema)->ostream instead of a member writeMetadata would clean up a bunch of uses.